docs/user_guide/ml_code.py

```diff
@@ -0,0 +1,289 @@
+# coding: utf-8
```

Comment:
Might make sense to move this to a dedicated example directory, so users can also find and access this example

Reply:
done, moved ml_code.py to an dir called example. Will switch all links to this file to the new directory in the next commit.